### PR TITLE
Bugfix FXIOS-4398 - "Find in Page" search bar text don't respect darktheme

### PR DESCRIPTION
**Pull request by dicarobinho**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-4398)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/11026)

## :bulb: Description
Changed colors for **Find in Page Search Bar**
Set render mode for **Find in Page Search Bar** buttons to template.

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: firefox-ios/Client/Frontend/Browser/FindInPageBar.swift [hunk 1]
```diff
@@ -50,21 +50,27 @@ class FindInPageBar: UIView, ThemeApplicable {
...
-        button.setImage(UIImage(named: StandardImageIdentifiers.Large.chevronUp), for: .normal)
+        button.setImage(
+            UIImage(named: StandardImageIdentifiers.Large.chevronUp)?.withRenderingMode(.alwaysTemplate),
+            for: .normal)
         button.accessibilityLabel = .FindInPagePreviousAccessibilityLabel
         button.addTarget(self, action: #selector(self.didFindPrevious), for: .touchUpInside)
         button.accessibilityIdentifier = AccessibilityIdentifiers.FindInPage.findPreviousButton
     }
-
+
```
Comment (nbhasin2) [inline firefox-ios/Client/Frontend/Browser/FindInPageBar.swift:60]:
```suggestion
```

@dicarobinho it seems like this PR has lint issue

I suggest installing `swiftlint` using homebrew and running it before pushing any code 

image

**Review by nbhasin2 — CHANGES_REQUESTED** [review]

File: firefox-ios/Client/Frontend/Browser/FindInPageBar.swift (hunk 1 above)
Comment (dicarobinho) [reply firefox-ios/Client/Frontend/Browser/FindInPageBar.swift]: @nbhasin2 is seems like on line 68 is the same lint issue. May you have please a quick change suggestion like this one for that line?

File: firefox-ios/Client/Frontend/Browser/FindInPageBar.swift [hunk 2]
```diff
@@ -50,21 +50,26 @@ class FindInPageBar: UIView, ThemeApplicable {
...
-        button.setImage(UIImage(named: StandardImageIdentifiers.Large.chevronDown), for: .normal)
+        button.setImage(
+            UIImage(named: StandardImageIdentifiers.Large.chevronDown)?.withRenderingMode(.alwaysTemplate),
+            for: .normal)
         button.accessibilityLabel = .FindInPageNextAccessibilityLabel
         button.addTarget(self, action: #selector(self.didFindNext), for: .touchUpInside)
         button.accessibilityIdentifier = AccessibilityIdentifiers.FindInPage.findNextButton
     }
-
+
```
Comment (nbhasin2) [inline firefox-ios/Client/Frontend/Browser/FindInPageBar.swift:68]:
```suggestion
```

**Review by cyndichin — APPROVED** [review]
Color updates looks good on all modes and iPad / iPhone! Thanks! 

**iPhone**
## Screenshots 
| Light | Dark | Private |
| --- | --- | --- |
| Simulator Screenshot - iPhone 15 Pro - 2024-03-26 at 09 39 06 | Simulator Screenshot - iPhone 15 Pro - 2024-03-26 at 09 38 39 | Simulator Screenshot - iPhone 15 Pro - 2024-03-26 at 09 38 19 |


**iPad**
| Light | Dark | Private |
| --- | --- | --- |
| simulator_screenshot_FBC41C96-6F03-4453-A8CA-A584D6528B56 | simulator_screenshot_5E82D210-F560-49F6-8C34-B6A873DFF54C | simulator_screenshot_29F4ED97-8CF0-4AD9-8E1E-FAF20133C25F |

**Comment by cyndichin** [comment]
separate from the color changes, but also notice that some padding and close button look a bit off compared to Figma file. Not sure if this is something we want to refine as well. cc: @cwzilla

**Comment by cwzilla** [comment]
> separate from the color changes, but also notice that some padding and close button look a bit off compared to Figma file. Not sure if this is something we want to refine as well. cc: @cwzilla

That was actually just a quick mockup to convey the colours/tokens to use, so it's not accurate! We can stick with what's already implemented.

**Review by nbhasin2 — APPROVED** [review]

**Comment by nbhasin2** [comment]
@mergify backport release/v125

**Pull request merged**